recipe/meta.yaml

```diff
@@ -40,6 +38,7 @@ test:
   imports:
     - jupyterlab_server
   commands:
+    - pip check
```

Comment:
and this line does not do the same as line below ?

Reply:
ah, it is the same indeed. I did not notice it. I will remove mine. Thanks!